src/Compilers/CSharp/Portable/LanguageVersion.cs

```diff
@@ -69,6 +69,11 @@ public enum LanguageVersion
         CSharp7 = 7,
 
         /// <summary>
+        /// C# language version 7.1
+        /// </summary>
+        CSharp7_1 = 71,
```

Comment:
I did not get that impression. Anyways, it is easy to change if it ends up different. I'll leave it as is.
The biggest cost for the change would be rebaselining the langversion tests and we will need to do that nayways since it now literally prints `71`, which does not seem right whatever way :-)